app/database_etl/location_utils/location_util_functions.py

```diff
@@ -90,30 +87,21 @@ def get_city(row):
         # associate the city with the state
         # so that we can get a pin for it
         if row['state'] and len(row['state']) == 1:
-            cities_return = []
-            for city in cities:
-                country_code = get_country_code(row['country'], iso3=False)
-                if country_code:
-                    cities_return.append(f"{city},{row['state'][0]}_{country_code}")
-                else:
-                    cities_return.append(f"{city},{row['state'][0]}")
-            return cities_return
+            return [f"{city},{row['state'][0]}" for city in cities]
         return cities
     else:
         return row['city']
 
-
-# Returns "state_countryCode"
-# Needed because country code is used to limit mapbox API
-# geosearch queries (improving query accuracy)
-def add_country_code_to_state(row):
-    if row['state']:
-        states = []
-        for state in row['state']:
-            country_code = get_country_code(row['country'], iso3=False)
+# stringlist ==> ["s1", "s2"]
+# returns ==> ["s1_countryCode", "s2_countryCode"]
+def add_country_iso2_to_stringlist(stringlist, country_name):
```

Comment:
having a hard time picturing - what is string list? a list of what kinds of strings? can you make this param more descriptive? at this point it could be like a list of ['table', 'chair', 'person'].... lmao

Reply:
practically its either a list of city strings or a list of state strings

Reply:
reason why I made this function accept a "stringlist" is because theoretically you could use this function on any arbitrary list of strings, like it would work for ['table', 'chair', 'person']. Just practically, we only care about using it on a list of city or state strings